zk/src/archive/web/js/zk/widget.js

```diff
@@ -2700,6 +2700,20 @@ function () {
 		return this;
 	},
 
+	/** A private to let widget re-render later.
+	 * It only works for widget call the re-render in the setAttrs phase,
+	 * avoiding widget re-rendering twice or more in setAttr phase.
+	 * Widget would do re-render right after it has done all the cmds from server(since ZK 8.6).
+	 * If it is not in setAttr phase, it would do re-render immediately.
+	 * @return zk.Widget this widget.
+	 */
+	_rerenderLater: function () {

```

Comment:
It means we don't want other widgets to call `wgt._rerenderLater` (though it can in JS)

In this situation I suggest `rerenderLater_` is better. Because the other widget class use it, not Widget class itself.